src/styleguide/_patterns/02-molecules/02-locations/location-card.mustache

```diff
@@ -0,0 +1,17 @@
+<div class="grav-c-location-card">
+  <div class="grav-c-location-card__img">
+    <img src="{{location.img}}" width="425" height="300" alt="">
+  </div>
+  <div class="grav-c-location-card__content">
+    <h2>{{location.name}}</h2>
+    <p>{{#location.addressline1}}{{location.addressline1}}<br>{{/location.addressline1}}
```

Comment:
do we need all these checks? Can we _safely_ assume they might not be there? Or can we assume that some are always there?

Reply:
Because we don't want the `<br>` to render if it's empty